packages/one-app-runner/src/startApp.js

```diff
@@ -0,0 +1,89 @@
+const { spawn } = require('child_process');
+const path = require('path');
+const fs = require('fs');
+
+module.exports = async function startApp({
+  moduleMapUrl,
+  rootModuleName,
+  modulesToServe,
+  appDockerImage,
+  envVars,
+  outputFile,
+  parrotMiddlewareFile,
+  devEndpointsFile,
+}) {
+  const generateEnvironmentVariableArgs = (vars) => {
+    const environmentVariablesWithProxyAdditions = {
+      ...vars,
+      ...process.env.HTTP_PROXY && { HTTP_PROXY: process.env.HTTP_PROXY },
+      ...process.env.HTTPS_PROXY && { HTTPS_PROXY: process.env.HTTPS_PROXY },
+      ...process.env.NO_PROXY && { NO_PROXY: process.env.NO_PROXY },
```

Comment:
maybe re order to allow for user override ?
```suggestion
      ...process.env.HTTP_PROXY && { HTTP_PROXY: process.env.HTTP_PROXY },
      ...process.env.HTTPS_PROXY && { HTTPS_PROXY: process.env.HTTPS_PROXY },
      ...process.env.NO_PROXY && { NO_PROXY: process.env.NO_PROXY },
      ...vars,
```

Reply:
I dont think we want the user to be able to override. We want to have the system env vars be the source of truth for proxy. Similar to how other tools do it. The `envVars` config should just be for env vars you want to pass to the one app runtime.

Technically the proxy env vars are also being passed to the one app runtime but for some reason I keep thinking it is best to keep the proxy env vars separate from the env vars that are more about configuring one app. Thoughts?

Reply:
yeah i like the idea of keeping the separate.